src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -2185,14 +2187,14 @@ internal IOperation CreatePropertySubpatternMember(Symbol symbol, ITypeSymbol ma
                     {
                         var constantValue = field.GetConstantValue(ConstantFieldsInProgress.Empty, earlyDecodingWellKnownAttributes: false);
                         var receiver = new InstanceReferenceOperation(
-                            InstanceReferenceKind.PatternInput, _semanticModel, nameSyntax, matchedType, constantValue, isImplicit: true);
+                            InstanceReferenceKind.PatternInput, _semanticModel, nameSyntax, matchedType, isImplicit: true);
```

Comment:
This change is a bug in the original implementation: the pattern input does not have a constant value.